apps/web/ui/links/link-card.tsx

```diff
@@ -14,7 +16,7 @@ export function LinkCard({ link }: { link: ResponseLink }) {
       <AddEditLinkModal />
       <CardList.Card
         key={link.id}
-        onClick={() => setShowAddEditLinkModal(true)}
+        onClick={isMobile ? undefined : () => setShowAddEditLinkModal(true)}
```

Comment:
Disabling clicking on the entire card for mobile - we could probably do something similar to the `modal-backdrop` checks instead, but I feel like with some of the small touch targets we have on mobile, this removes more potential frustration.

Reply:
okay sounds good!